### PR TITLE
Updated with Sampler

### PR DESCRIPTION
**Pull request by digvijaysingh1707**

### CODE REVIEW

File: trainer.py
Comment (harshagrawal13) [inline trainer.py:1]: Why did you comment this out?

File: troubleshoot_2.ipynb
Comment (harshagrawal13) [inline troubleshoot_2.ipynb:1]: Why did you commit a notebook?

File: config/trainer_config.json
Comment (harshagrawal13) [inline config/trainer_config.json:1]: Why is there a batch_value if there is already batch_size?

File: troubleshoot_2.ipynb
Comment (digvijaysingh1707) [reply troubleshoot_2.ipynb:1]: Not able to find the write value for iterations

**Pull request closed without merging**